### PR TITLE
Add meta description

### PR DESCRIPTION
**Pull request by ricardobeat**
Used by bookmark services and search engines.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -3,6 +3,7 @@
 <head>
     <meta charset="utf-8">
     <meta name="viewport" content="width=480">
+    <meta name="description" content="Tota11y is an accessibility toolkit that helps visualize how your site performs with assistive technologies.">
```
Comment (jdan) [inline index.html]: Can we downcase `Tota11y`? And maybe remove the period at the end as well.

**Comment by jdan** [comment]
Hi Ricardo! Thanks for this, I hadn't realized the importance of a meta description.

Mind [signing the CLA](http://khanacademy.org/r/cla) per the instructions in [CONTRIBUTING.md](https://github.com/Khan/tota11y/blob/master/CONTRIBUTING.md) so I can merge this in?

**Comment by jdan** [comment]
ping @ricardobeat

**Comment by ricardobeat** [comment]
Hi Jordan. I have to say I am not completely comfortable with signing the CLA - I don't know the exact legal implications for some of those terms, and don't have the time to investigate. Feel free to push a commit of your own based on this change, I'm happy to have helped.

**Comment by jdan** [comment]
Okay no worries! Please note that CLAs are fairly common, and you'll need to sign one to contribute to open source projects by Google, Facebook, Twitter, etc.

Sorry to have scared you off - happy to answer any questions.

**Pull request closed without merging**